arwen/common.go

```diff
@@ -3,19 +3,27 @@ package arwen
 type StorageStatus int
 
 const (
-	StorageUnchanged StorageStatus = 0
-	StorageModified  StorageStatus = 1
-	StorageAdded     StorageStatus = 3
-	StorageDeleted   StorageStatus = 4
+	StorageUnchanged StorageStatus = iota
```

Comment:
iota only for the first, the rest can be left unassigned. it does automatically.

Reply:
That was the first thing I tried. Got this:
```
arwen/common.go:7:19: const declaration cannot have type without expression
```

Reply:
Nevermind, the problem was the remaining datatype. Fixed.